### PR TITLE
Added note to readme, that nodejs/npm is needed for spotless to run

### PR DESCRIPTION
**Pull request by reckseba**

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ brew install openjdk@21
 brew install --cask docker # or just `brew install docker` if you don't want the Desktop app
 ```
 
+Install node for Spotless with Prettier being able to run.
```
Comment (hannes-mk) [inline backend/README.md]: Works for me.

Two minor ideas:

- The expression "... for Spotless with Prettier being able to run" could be simplified by using active voice: " ... for running Spotless with Prettier".
- We generally aim for the latest LTS version of node; we could mention that, here: "Install the latest LTS version of node for ...".

**Review by hannes-mk — APPROVED** [review]
Welcome to the code base! :wave:

File: backend/README.md (hunk 1 above)
Comment (reckseba) [reply backend/README.md]: Thanks for the advise! I committed the change.

**Comment by reckseba** [comment]
Close because commit were not signed.

**Pull request closed without merging**